hvantk/tests/test_gwas_coloc.py

```
@@ -0,0 +1,156 @@
+# ---------------------------------------------------------------------------
+# CLI
+# ---------------------------------------------------------------------------
+
+
+def test_gwas_coloc_cli_requires_n_for_finemap():
+    from hvantk.tools.qtl.qtlcascade_cli import qtlcascade_group
+
+    r = CliRunner().invoke(
+        qtlcascade_group,
+        ["gwas-coloc", "--endpoint", "I9_AF", "--chrom", "10", "--lead", "73600000",
+         "--eqtl", "QTD000251", "-o", "/tmp/out"],  # fine-map default on, no -n
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp/out"

(S108)